### DISCUSSION TITLE
Iped.exe: Could not find iped.jar file

### DISCUSSION THREAD

**Discussion by bitstream111** [sepinf-inc/IPED#335] [Q&A] [answered]
I am running it on windows, installed dependencies (Maven, Sleuthkit). When i run it through command line, it gives error that could not access iped.jar file or iped-searchapp.jar file and these files are not available in package.
Can you please guide if i am missing something in configuration. I have no idea how to run it.

**Comment by kraftdenker** [comment]
Hello,

Check if there is the file *iped-searchapp.jar *inside the
path [where_you_installed]*\IPED\target\release\iped-*[iped_verison_number]
*\lib*

Could you inform us the IPED version?



Am Do., 10. Dez. 2020 um 06:15 Uhr schrieb bitstream111 <
notifications@github.com>:

> I am running it on windows, installed dependencies (Maven, Sleuthkit).
> When i run it through command line, it gives error that could not find
> iped.jar file or iped-searchapp.jar file.
> Can you please guide if i am missing something in configuration. I have no
> idea how to run it.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sepinf-inc/IPED/issues/335>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFVAK4V4LJTF4B7G42XD2RDSUCGUNANCNFSM4UUW6SEQ>
> .
>


-- 
A.M

**Comment by lfcnassif** [comment, marked as answer]
Probably you downloaded the source but did not compile it. Please see the compile instructions on github first page or download a compiled version from https://github.com/sepinf-inc/IPED/releases

**Comment by bitstream111** [comment]
> Probably you downloaded the source but did not compile it. Please see the compile instructions on github first page or download a compiled version from https://github.com/sepinf-inc/IPED/releases

Appreciated, It is working now.

**Reply by FessyD1993** [reply]
Hello there at bitstream111, i am researching in the field of digital forensics and i am new to the tool 
i am choosing this tool because it was recommended that i should use it as it tells a lot about almost everything that is forensically sound
